src/getting-started/content-settings.md

```diff
@@ -0,0 +1,36 @@
+# Changing Content Settings
+
+Adjusting your content settings to "Mature" or "Adult" is often necessary to add content to your app. Here is how to do it:
+
+1. Visit the [Paperback Portal](https://portal.paperback.moe).
+2. Log in with your Paperback Account or create one if you don't have one.
+3. Customize the content setting to your preference (**Restricted**, **Mature** or **Adult**).
+4. If you were logged in, log out of the app.
+5. Log back in again.
```

Comment:
```suggestion
1. Visit the [Paperback Portal](https://portal.paperback.moe).
2. Log in with your Paperback Account or create one if you don't have one.
3. Customize the content setting to your preference (**Restricted**, **Mature** or **Adult**).
4. If you were logged in, log out of the app.
5. Log back in again.
```

Reply:
Could you elaborate on why you removed this? I personally think it is important to tell users why to do something.

Reply:
The phrasing is misleading. Changing content settings is only "necessary" when you're adding content that doesnt match your existing content setting. Besides that, the sentence also provides no valuable information it is better to get straight to the point with the steps to change the content setting.